internal/integration/knot/network/subscriber.go

```diff
@@ -3,9 +3,9 @@ package network
 const (
 	queueName = "chirpstack-knot-messages"
 
-	bindingKeyRegistered    = "device.registered"
-	bindingKeyUnregistered  = "device.unregistered"
-	bindingKeyUpdatedConfig = "device.config.updated"
+	BindingKeyRegistered    = "device.registered"
```

Comment:
This is a exported const. These constants should have comment.

Reply:
Ok.